scripts/build_transport_demand.py

```diff
@@ -181,7 +181,7 @@ def bev_dsm_profile(snapshots, nodes, options):
 
     snapshots = pd.date_range(freq="h", **snakemake.config["snapshots"], tz="UTC")
 
-    Nyears = 1
+    nyears = len(snapshots) / 8760
```

Comment:
I think this has to use the snapshot weightings, right?

Reply:
Does that make a difference? snapshot weighting are always summing up to number of hours from start to end or?

Reply:
ah, this is before time aggregation. You're right.